zoo-redis/custom-entrypoint.sh

```
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+# Set up Redis configuration directory
+mkdir -p /usr/local/etc/redis
+
+# Dynamically generate Redis configuration and ACL files here, using environment variables
+echo "aclfile /usr/local/etc/redis/custom_aclfile.acl" > /usr/local/etc/redis/redis.conf
+
+# Generate ACL file using environment variables
+if [ -n ${REDIS_USERNAME} ] && [ -n ${REDIS_PASSWORD} ]; then
```

Shellcheck

[error] -n doesn't work with unquoted arguments. Quote or use [[ ]].

(SC2070)